scripts/specs.ts

```diff
@@ -96,6 +96,10 @@ const defaultAllowlist: allowlistItem[] = [
     [
         "https://immersive-web.github.io/webvr/spec/1.1/",
         "Allowed because this is the legacy spec that defines WebVR."
+    ],
+    [
+        "https://github.com/tc39/proposal-import-attributes/tree/abca60286360b47f9a6be25a28f489c2cb157beb",
+        "Allowed because import assertions were replaced in-place by import attributes. Remove this exception when javascript.statements.import.import_assertions is dropped from BCD in June 2026."
```

Comment:
Nice-to-have: a script that goes over this array and checks if the specs listed here are actually used in features. This way we don't need to remember to remove items.